neurots/generate/algorithms/tmdgrower.py

```diff
@@ -61,6 +65,12 @@ def __init__(
         self.apical_section = None
         self.apical_point_distance_from_soma = 0.0
         self.persistence_length = self.barcode.get_persistence_length()
+        if pia_direction is None:
+            self.pia_rotation = np.eye(3)
+        else:
+            self.pia_rotation = rotation.rotation_matrix_from_vectors(
+                PIA_DIRECTION, pia_direction
+            ).T
```

Comment:
Would it make sens to move this in `AbstractAlgo.__init__` to avoid duplication?

Reply:
I was thinking not to, so that growers can completely avoid using this (like astrocytes)